apis/v1alpha1/validation.go

```diff
@@ -21,6 +21,10 @@ import (
 	"net/mail"
 )
 
+const (
+	AssetInfoKey = "assetInfo"
```

Comment:
Go doesn't have enum so the convention is to put "type" as the first part of the variable name, e.g. `AnnotationKeyAssetInfo`, `AnnotationKeyOtherStuff`

Reply:
Thanks for these comments, I will keep them in-mind! Already updated.